### ISSUE TITLE
sigmoid / WScaleLayer

### ISSUE THREAD

**Issue by elbaro** [zhangqianhui/progressive_growing_of_gans_tensorflow#1]
Hi, I am glad to find the implementation using wgan-gp.

Can you explain the differences from the reference implementation:
- sigmoid at the end of Discriminator
- No use of WScaleLayer

**Comment by zhangqianhui** [comment]
* I don't use the sigmoid for the loss function.  You can check it.
* I implement WScalelayer for learning rate equalization , but get the bad performance. If you implement it with good performance, please make a PR. Thank you.

**Comment by elbaro** [comment]
> I implement WScalelayer for learning rate equalization , but get the bad performance.

That's good to know. Thanks!
